tests/acceptance/features/webUIMoveFilesFolders/moveFolders.feature

```diff
@@ -68,7 +68,7 @@ Feature: move folders
       | "question?" | "target-folder-with-question?" |
       | "&and#hash" | "target-folder-with-&and#hash" |
 
-  @issue-ocis-1328
+  @skipOnOCIS
```

Comment:
This test is not related to https://github.com/owncloud/ocis/issues/1328, was linked to it in expected to fail. But after share jail the shares reside in a different section.